js/bundle/src/cli.ts

```diff
@@ -37,55 +38,82 @@ export function addFilesRecursively(
   }
 }
 
-export function main() {
-  const options = commander
+function readOptions() {
+  return commander
     .requiredOption('-d, --dir <directory>', 'input root directory (required)')
     .option('-b, --baseURL <URL>', 'base URL')
     .option(
       '-f, --formatVersion <formatVersion>',
-      'webbundle format version, possible values are "b1" and "b2" (default: "b2")',
-      'b2'
+      'webbundle format version, possible values are ' +
+        APPROVED_VERSIONS.map((v) => `"${v}"`).join(' and ') +
+        ' (default: "' +
+        DEFAULT_VERSION +
+        '")',
+      DEFAULT_VERSION
     )
```

Comment:
Have you looked into using the `.choices` function here? [From the commander docs](https://www.npmjs.com/package/commander):

```ts
  .addOption(new Option('-d, --drink <size>', 'drink size').choices(['small', 'medium', 'large']))
```

Reply:
I feel like this is a functional change so might look into this a bit later if i have time